velox/dwio/dwrf/test/TestReader.cpp

```diff
@@ -505,6 +505,47 @@ TEST(TestReader, testStatsCallbackFiredWithFiltering) {
   EXPECT_EQ(selectedKeyStreamsAggregate, 4);
 }
 
+TEST(TestReader, testEstimatedSize) {
+  const std::string fmSmall(getExampleFilePath("fm_small.orc"));
+
+  std::shared_ptr<const RowType> requestedType =
+      std::dynamic_pointer_cast<const RowType>(HiveTypeParser().parse("struct<\
+          id:int,\
+      map1:map<int, array<float>>,\
+      map2:map<string, map<smallint,bigint>>,\
+      map3:map<int,int>,\
+      map4:map<int,struct<field1:int,field2:float,field3:string>>,\
+      memo:string>"));
+
+  ReaderOptions readerOpts{defaultPool.get()};
+
+  {
+    RowReaderOptions rowReaderOpts;
```

Comment:
nit: move rowReaderOpts creation to where it is first used? l531 when we create rowReader